src/screens/Principal/index.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+import { fetchGame } from '../../services/gameService';
+
+import Header from './components/Header';
+import Blocks from './components/Blocks';
+import BoardGame from './components/BoardGame';
+import Resolution from './components/Resolution';
+import ImgBackground from './components/ImgBackground';
+import styles from './index.module.scss';
+
+class Principal extends React.Component {
+  state = {
+    transactions: {},
+    miner: {},
+    disabled: true
+  };
+
+  getGame = async () => {
+    const res = await fetchGame();
+    const { transactions, miner } = res.data;
+
+    this.setState({ transactions, miner });
+  };
+
+  componentDidMount() {
+    this.getGame();
+  }
+
+  handleDisabledButton = ({ disabled }) => this.setState({ disabled });
+
+  render() {
+    return (
+      <div className={styles.mainContainer}>
+        <Header name={this.state.miner.name} />
```

Comment:
Aquí podrías utilizar la desestructuración para el this.state. Veo que lo usaste en varios lugares, eso está genial!

```suggestion
  render() {
  const { miner, transactions, disabled } = this.state;
    return (
      <div className={styles.mainContainer}>
        <Header name={miner.name} />
```